app/src/main/java/com/github/se/cyrcle/model/parking/ParkingViewModel.kt

```diff
@@ -82,10 +82,21 @@ class ParkingViewModel(
   private val _selectedParking = MutableStateFlow<Parking?>(null)
   val selectedParking: StateFlow<Parking?> = _selectedParking
 
+  private val _selectedImage = MutableStateFlow<String?>(null)
```

Comment:
Just for consistency could you also add documentation here because the difference between `selectedImage` and `selectedParking` image is a bit confusing.